x-pack/packages/kbn-elastic-assistant/impl/assistant/settings/evaluation_settings/translations.ts

```diff
@@ -78,6 +78,36 @@ export const CONNECTORS_LABEL = i18n.translate(
   }
 );
 
+export const EVALUATOR_MODEL = i18n.translate(
```

Comment:
nit: Evals is always developer only, right? Never user facing? We probably don't need to be using i18n resources

Reply:
Yes, the audience of evals is developers, who must enable this UI via the `assistantModelEvaluation` feature flag. The original author of this file also used `i18n`, so the new entries are consistent with that style.

Reply:
It's a nit so no big deal, but it might be nice to remove this pattern to avoid unnecessary resource use